tests/unit_tests/gui/test_main_window.py

```diff
@@ -60,53 +64,134 @@ def opened_main_window(source_root, tmpdir_factory, request):
 
 @pytest.mark.usefixtures("use_tmpdir")
 @pytest.fixture(scope="module")
-def esmda_has_run(opened_main_window, request):
-    qtbot = QtBot(request)
-    gui = opened_main_window
-    qtbot.addWidget(gui)
-
+def esmda_has_run(opened_main_window, run_experiment):
```

Comment:
`opened_main_window` not used I think.

Reply:
True, but was needed, moved it to: `usefixture` instead